Dockerfile

```diff
@@ -138,6 +138,7 @@ COPY --from=build-texlive /texlivebins /usr/local/texlive
 # Copy only the fonts we're using from the template.
 COPY --from=build-fonts \
     /usr/share/fonts/truetype/msttcorefonts/Arial* \
+    /usr/share/fonts/truetype/msttcorefonts/Courier* \
```

Comment:
How important is it to specifically use Courier? We use Noto Sans Mono for code (`monospaced`) text currently. However, TCG's templates haven't historically been that picky about monospaced fonts.

Reply:
for now, we need this to be in Courier as all of TCG storage spec is Courier. We can remove this in future if we decide to not use courier.

Reply:
Yup, changeable down the road easily.